README.md

```diff
@@ -66,9 +95,9 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
+[X] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
+[X] `$(".post").html()`
 [] `document.querySelectorAll(".post")[0].innerHTML`
```

Comment:
Yes, but this question is tricky because technically this would work as well: 
`[x] document.querySelectorAll(".post")[0].innerHTML`

Although jQuery is loaded, this Vanilla JS would be valid